lib/kube/proxy/server.go

```diff
@@ -74,6 +78,7 @@ type TLSServerConfig struct {
 	OnReconcile func(types.KubeClusters)
 	// CloudClients is a set of cloud clients that Teleport supports.
 	CloudClients cloud.Clients
+	AWSClients   *awsClientsGetter
```

Comment:
if this is a private structure w/out any way of building it, does it make sense for the field to public?

Reply:
It was to be consistent with the rest, but indeed, not needed. Made private.